src/pydata_sphinx_theme/assets/styles/sections/_sidebar-secondary.scss

```diff
@@ -24,13 +24,35 @@
   }
 }
 
-.bd-toc {
-  -ms-flex-order: 2;
-  order: 2;
-  max-height: calc(100vh - 5rem);
+.bd-sidebar-secondary {
+  z-index: $zindex-modal;
+  top: 0;
+  min-height: 100vh;
+  max-height: 100vh;
+  border-left: 1px solid var(--pst-color-border);
+  background-color: var(--pst-color-background);
+  padding-top: 1em;
   overflow-y: auto;
-  top: var(--pst-header-height);
-  position: sticky;
+  display: flex;
+  flex-direction: column;
+  position: fixed;
+  right: 0;
+
+  // hide the sidebar in narrow screens
+  // based on the toc_show_width variable
+  @each $break, $width in $grid-breakpoints {
+    &.col-#{$break}-2 {
+      @include media-breakpoint-up($break) {
```

Comment:
I'm a bit confused by this - it looks like we are adding the same `@include` breakpoint logic to every possible breakpoint, but shouldn't we only need to do this for the smallest breakpoint and then it applies to all widths above it?

Reply:
So yes I want the same behavior for each break. But what I want in fact is to have the same behaviour whatever the value set in: https://github.com/pydata/pydata-sphinx-theme/blob/26554f2bceb856c78a22e00677892a3d58ceb201/src/pydata_sphinx_theme/theme/pydata_sphinx_theme/sections/sidebar-secondary.html#L4. only one break will be selected per page.